plugins/blockvolume/hostvol/utils.go

```diff
@@ -15,34 +15,70 @@ import (
 	"github.com/gluster/glusterd2/glusterd2/volume"
 	"github.com/gluster/glusterd2/pkg/api"
 	"github.com/gluster/glusterd2/pkg/size"
-
 	log "github.com/sirupsen/logrus"
 )
 
+func filterSize(volinfo *volume.Volinfo, size uint64) bool {
```

Comment:
size filter is already present, please use the existing one.

Reply:
Done